### DISCUSSION TITLE
Github Pages 404

### DISCUSSION THREAD

**Discussion by izzzapr** [community/community#54657] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

# Hello,

I just published my Github pages ![site](timnothere.github.io) and its giving me a 404 error. how do i fix this?

**Comment by radar-blip** [comment]
Changing `Index.html` to`index.html `might help, but im not sure.
I think it has to be lowercase.

**Reply by izzzapr** [reply]
yes that was the error, thank you!

**Reply by radar-blip** [reply]
You're welcome. Glad to help!
